_rules/device-motion-user-interface-7677a9.md

```diff
@@ -0,0 +1,191 @@
+---
+id: 7677a9
+name: Device motion based functionality can also be operated from the user interface
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by device motion can also be operated by user interface components.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'    
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available supporting the same outcomes of the event.
+
+## Assumptions
+
+- The motion to operate the device is not used through an [accessibility supported][] interface, which is listed as a valid exception to SC 2.5.4.
+- The motion is not [essential][] for the functionality it triggers, which is listed as a valid exception to SC 2.5.4.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This [HTML document][] that can be operated through the device's orientation to increase and decrease the value of a slider has the same functionality available through `button` elements.
```

Comment:
"instrument" is defined as "an HTML element" (https://github.com/act-rules/act-rules.github.io/pull/960/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md#diff-765403c47c39874f2af13d4ec2e178aaR8)

So, technically, the "instrument" is here the `div`, not the **two** `buttons`…

Maybe it's better (more flexible) to change the expectation and require "a set of instruments" or something similar…

(same on other examples)